salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -1,8 +1,14 @@
-{# todo: mass redirects are useful in general. lets find a way of supporting this that isn't elife-specific #}
-{% if pillar.elife.domain == 'elifesciences.org' %}
-    include /etc/nginx/traits.d/redirect-existing-paths.conf;
+{% if pillar.journal.get('default_host') %}
+    {% set public_host = pillar.journal.get('default_host') %}
+{% elif salt['elife.only_on_aws']() %}
```

Comment:
this should be backward compatible as `end2end` where `default_host` isn't defined yet would fall back to `end2end--journal.elifesciences.org`, which is the non-CDN hostname but better than nothing